std/internal/math/gammafunction.d

```diff
@@ -524,6 +524,14 @@ real logGamma(real x)
     return q ;
 }
 
+version (NetBSD)
+{
+    unittest
+    {
+        pragma(msg, "NetBSD does not support functions for real type");
+    }
+}
```

Comment:
This doesn't actually make sense. If it's a pragma, there's no reason to put it in a unittest block - just put it in `version(unittest)`, and having it print over and over again like this doesn't make sense with a pragma (you have the exact same pragma in several places). If you want to print it with the unit test, use `writeln`. If you want it as a pragma, then at least put the function that it goes with in the message so that they're not all the same. What's there is simply too generic for a message that gets printed on compilation and doesn't have any file or line number associated with it. And actually, if these functions aren't supported on NetBSD, shouldn't they just be versioned out entirely? What's the point of versioning out tests for functions but keeping the functions in?